mnefun/_mnefun.py

```diff
@@ -2419,21 +2443,26 @@ def do_preprocessing_combined(p, subjects, run_indices):
                        **old_kwargs)
             raw.add_proj(projs)
             raw.apply_proj()
-            pr, eog_events = \
-                compute_proj_eog(raw, n_grad=proj_nums[1][0],
-                                 n_jobs=p.n_jobs_mkl,
-                                 n_mag=proj_nums[1][1], n_eeg=proj_nums[1][2],
-                                 tmin=eog_t_lims[0], tmax=eog_t_lims[1],
-                                 l_freq=None, h_freq=None, no_proj=True,
-                                 ch_name=p.eog_channel,
-                                 reject=p.ssp_eog_reject, average=p.proj_ave)
-            if eog_events.shape[0] >= 5:
-                write_events(eog_eve, eog_events)
+            eog_events = find_eog_events(
+                raw, ch_name=p.eog_channel, reject_by_annotation=True)
+            eog_epochs = Epochs(
```

Comment:
Yes, because if there is a segment of time that is BAD (i.e., really noisy) you probably don't want it contaminating your estimate of the artifact waveform that will be used for projection over the clean data.